riak_test/clocksi_test.erl

```diff
@@ -19,20 +19,40 @@
 %% -------------------------------------------------------------------
 -module(clocksi_test).
 
--export([confirm/0, clocksi_test1/1, clocksi_test2/1,
-         clocksi_test_read_wait/1, clocksi_test4/1, clocksi_test_read_time/1,
-         clocksi_test_certification_check/1,
-         clocksi_multiple_test_certification_check/1, spawn_read/3, clocksi_test_prepare/1, spawn_com/2]).
+-export([confirm/0,
+	 clocksi_test1/1,
+	 clocksi_test2/1,
+	 clocksi_test3/1,
+	 clocksi_test5/1,
+         clocksi_test_read_wait/1,
```

Comment:
It's probably from people using different editors.  I miss these because they look fine in the raw file
https://raw.githubusercontent.com/SyncFree/antidote/materializer_multidc_eval_strong/riak_test/clocksi_test.erl

I think when you press the "fix-indenting" button in emacs it just fills up the missing spaces to make it even with the other lines.  So with people using different editors and when some spaces are already there you get a mix of tab and space characters.  I fix them when I notice them.